github/free-disk-space/.github/workflows/test.yml

```diff
@@ -0,0 +1,20 @@
+name: Free Disk Space (Ubuntu)
+on: push
+
+jobs:
+  free-disk-space:
+    runs-on: ubuntu-latest
+    steps:
+      # If there is a problem with this GitHub Actions, this step will fail
+      - name: Free Disk Space
+        uses: jlumbroso/free-disk-space@main
```

Comment:
update this?

Reply:
This workflow will not run, is jsut cloned from the source repo.

The idea is to get the repo as 1:1 so if we want to update it we can simply just get the update it in-place.